docs/user/canonical-urls.rst

```diff
@@ -0,0 +1,55 @@
+Canonical URLs
+==============
+
+`Canonical URLs`_ allow you to specify the preferred version of a web page to prevent duplicated content.
+They are mainly used by search engines to link users to the correct
+version and domain of your documentation.
+
+.. _Canonical URLs: https://datatracker.ietf.org/doc/html/rfc6596
```

Comment:
We definitely don't want to link to this. Perhaps https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls?